extensions/escape-game/standalone.mjs

```diff
@@ -552,8 +551,7 @@ class Standalone {
 
       this.finalizeTeam(playerID, this.playerToTeam.get(playerID).teamId);
 
-      await this.helper.callS2cAPI(playerID, 'dialog', 'showDialog', 60*1000,
-        sfInfo.name, 'Finalized');
+      // await this.helper.callS2cAPI(playerID, 'dialog', 'showDialog', 60*1000, sfInfo.name, 'Finalized');
```

Comment:
Is this intended?

Reply:
Yes, @ccali66 already create a finalized dialog in the team manager. This removes the redundant dialog.